easybuild/framework/easyconfig/tools.py

```diff
@@ -538,6 +539,16 @@ def review_pr(paths=None, pr=None, colored=True, branch='develop'):
         else:
             lines.extend(['', "(no related easyconfigs found for %s)\n" % os.path.basename(ec['spec'])])
 
+    labels = []
+    file_info = det_file_info(pr_files, download_repo_path)
+    if any(file_info['new_folder']):
+        labels.append('new')
+    if any(file_info['new_file_in_existing_folder']):
+        labels.append('update')
+
+    if labels:  # TODO: compare to actual PR labels?
```

Comment:
@migueldiascosta Makes sense to fix this first?

Reply:
Done... now thinking that since this is likely being run by a maintainer, in that case it could actually set the labels when they are missing...